packages/trpc/server/routers/viewer/auth/verifyToken.handler.ts

```diff
@@ -0,0 +1,37 @@
+import dayjs from "@calcom/dayjs";
+import { prisma } from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TVerifyTokenSchema } from "./verifyToken.schema";
+
+type VerifyTokenOptions = {
+  input: TVerifyTokenSchema;
+};
+
+export const verifyTokenHandler = async ({ input }: VerifyTokenOptions) => {
```

Comment:
this looks similar to packages/trpc/server/routers/viewer/organizations/verifyCode.handler.ts. we can maybe use verifycode.handler instead of creating new one and verificationToken table is used to store inviteTokens

Reply:
@rkreddy99  we don't need this function. we can use verifyCode.handler